src/tbb/CMakeLists.txt

```diff
@@ -71,6 +71,10 @@ if (NOT ("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "(armv7-a|aarch64|mips|arm64|riscv)
     target_compile_definitions(tbb PRIVATE __TBB_USE_ITT_NOTIFY)
 endif()
 
+if (TBB_VERIFY_DEPENDENCY_SIGNATURE)
+    target_compile_definitions(tbb PRIVATE __TBB_VERIFY_DEPENDENCY_SIGNATURE)
+endif()
```

Comment:
See target_compile_definitions call at line 58. You should be able to use the same pattern as for BUILD_SHARED_LIBS.

Reply:
Applied.